src/plugins/data_source_management/public/components/popover_button/popover_button.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import React from 'react';
+import { EuiButtonEmpty } from '@elastic/eui';
+import { i18n } from '@osd/i18n';
+
+interface DataSourceMenuPopoverButtonProps {
+  className: string;
+  label?: string;
+  onClick: () => void;
+}
+
+export const DataSourceMenuPopoverButton: React.FC<DataSourceMenuPopoverButtonProps> = ({
```

Comment:
nit: can we change the file name as well?

Reply:
yes